msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -133,13 +133,13 @@ With post-meeting app experience, you can view the results of the meeting, such
 
 The following image displays the **Contoso** tab with results of poll and feedback received from meeting attendees:
 
-# [Desktop](#tab/desktop)
+# [Desktop](#tab/desktop3)
 
-:::image type="content" source="~/assets/images/apps-in-meetings/post.png" alt-text="Contoso tab with results.":::
+:::image type="content" source="~/assets/images/apps-in-meetings/post.png" alt-text="The screenshot is an example that shows the Contoso tab with results in desktop.":::
```

Comment:
Wouldn't the same result be displayed in the web app? I think so and then the mention of desktop as a platform is incomplete without a mention of Web as a platform.

Reply:
It can be shared through mobile or desktop and not through web. Therefore, did not mention web.